cmd/prometheus/main.go

```
@@ -278,6 +278,11 @@ func main() {
 		os.Exit(2)
 	}
 
+	if _, err := config.LoadFile(cfg.configFile); err != nil {
```

A comment that we're doing this so we'll fail early would be useful.

---

Done. I hope that is satisfactory.